code/area.dm

```diff
@@ -1697,6 +1789,10 @@ area/station/hallway/starboardupperhallway
 	name = "Mining Magnet Control Room"
 	icon_state = "miningp"
 
+/area/station/mining/cargo_staff_room
+	name = "Cargo Staff Room"
+
+ABSTRACT_TYPE(/area/station/bridge)
```

Comment:
why abstract bridge?

Reply:
it's a top-level station area; mining, medbay, bridge, etc. do you think it should be non-abstract? as a general rule I tried to keep `/area/station/foo` abstract